automod2/automod2.py

```diff
@@ -26,7 +26,6 @@
 from .utils.dataIO import fileIO
 from .utils.settings import Settings
 
-

```

Comment:
pep8 says there should be two blank lines here.

I should probably enable a pep8 validation bot on commits.

Reply:
fixed i thought i had to add an import but i didnt gg